### DISCUSSION TITLE
Plotting Berkeley Earth data on an equal-area (unstructured?) grid

### DISCUSSION THREAD

**Discussion by mktippett** [pydata/xarray#8015] [Q&A] [answered]
When I do this
```
# data from https://berkeley-earth-temperature.s3.us-west-1.amazonaws.com/Global/Gridded/Complete_TAVG_Daily_EqualArea.nc
ds = xr.open_dataset('Complete_TAVG_Daily_EqualArea.nc')
plt.scatter(x=ds.longitude, y=ds.latitude, c=ds.temperature.isel(time=-1), s=1)
plt.show() 
```

I get 
![image](https://github.com/pydata/xarray/assets/31803994/9e2ce99a-f1a9-4047-a646-17a86d37113b)
which is nice. Is there a way to plot the data without dots? More like a pcolormesh plot? Or is there not enough description of the grid in the dataset?

Thanks!

**Comment by dcherian** [comment, marked as answer]
You'll have to grid the dataset with something like https://docs.scipy.org/doc/scipy/reference/generated/scipy.interpolate.griddata.html
